### DISCUSSION TITLE
Question regarding new modifiers for terminal in 0.10

### DISCUSSION THREAD

**Discussion by asmodeus812** [ibhagwan/fzf-lua#1145] [Q&A] [answered]
Hi @ibhagwan with the arival of 0.10, and the new support for `modifiers to :terminal`, such as support for `:botright terminal`, how would that affect the split option should we want to use this instead of using the current behaviour i.e. `:botright new` which splits, the current window / buffer and then i suppose fzf lua takes over loads/opens a termjob inside of that split instead.

**Comment by ibhagwan** [comment, marked as answer]
Hi @asmodeus812, I don't see any reason to use `botright terminal` , fzf-lua needs an empty buffer to work in so it can then run `termopen` inside it:
https://github.com/ibhagwan/fzf-lua/blob/213da0b65bf81e8849d1ebf2a3d47f32a7dc9000/lua/fzf-lua/fzf.lua#L249
https://github.com/ibhagwan/fzf-lua/blob/213da0b65bf81e8849d1ebf2a3d47f32a7dc9000/lua/fzf-lua/fzf.lua#L262

Opening a terminal window instead of a plain buffer has no advantage over `botright new`, in fact it might even fail `termopen` if I recall correctly.

**Comment by asmodeus812** [comment]
Sorry forgot about that one, yes indeed fzf lua is using termopen then it needs an empty unmodified buffer, we cant use :termianl. That seems to resolve my question. Thanks
